docs/src/pages/components/sliderfield/demo.tsx

```diff
@@ -1,40 +1,69 @@
 import * as React from 'react';
+import { SliderField, SliderFieldProps } from '@aws-amplify/ui-react';
+import { Demo } from '@/components/Demo';
+import { SliderFieldPropControls } from './SliderFieldPropControls';
+import { useSliderFieldProps } from './useSliderFieldProps';
+import { getPropString } from '../utils/getPropString';
+import { demoState } from '@/utils/demoState';
 
-import { Flex, SliderField } from '@aws-amplify/ui-react';
+const propsToCode = (props: SliderFieldProps) => {
+  return (
+    `<SliderField` +
+    `\n  label="${props.label}"` +
+    (props.min ? `\n  min={${props.min}}` : '') +
```

Comment:
can you use the getPropString helper function for more of these properties? looks like maybe min and step could use it and maybe we create a second util function that does the same thing for boolean flags isDisabled, isValueHidden, labelHidden.

Reply:
Good idea. `min`, `max` and `step` are all numbers though, so we can't use getPropString on them. I was thinking about making a general `propsToCode` util function, which is maybe a better longterm solution

Reply:
yeah we should probably just have one of these util functions for each of the common types, number, string, and boolean.

Reply:
`getPropString` takes a number or a string and returns a string, so you should be able to use it. I'm open to better names for the function though if it's confusing.

Reply:
If we used `getPropString` for a prop which is type `number`, then it would return something like:
`min="7"`

However, we'd want to show that prop set to a number, for example:
`min={7}`

I can adjust the `getPropString` function to handle both the number and string cases.